uast/uast.go

```diff
@@ -182,7 +182,11 @@ func AsPosition(m nodes.Object) *Position {
 }
 
 // PositionsOf returns a complete positions map for the given UAST node.
-func PositionsOf(m nodes.Object) Positions {
+func PositionsOf(n nodes.Node) Positions {
```

Comment:
Please update the comment to document that this is empty for a non-Object. Previously it was clear from the type signature, but no longer—and intuitively it's not obvious that an array wouldn't also have positions (e.g., the start and end of its span). Not saying it should, but if I asked for the position of an arbitrary node I would be surprised that it should't work for an Array.

Reply:
Done.